openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -344,6 +344,16 @@ def search_emails(self, request):
             }
         ]
         """
+        if not request.user.is_staff:
+            return Response(
+                {
+                    'developer_message': 'not_found',
+                    'user_message': 'Not Found'
+                },
+                status=status.HTTP_404_NOT_FOUND
+            )
+            raise Http404
```

Comment:
it will be better to add a comment here to explain why we are throwing 404 instead of 403

Reply:
this should be 403 ...